LT.DigitalOffice.WikiService/src/WikiService/Startup.cs

```diff
@@ -0,0 +1,208 @@
+using LT.DigitalOffice.Kernel.BrokerSupport.Configurations;
+using LT.DigitalOffice.Kernel.BrokerSupport.Extensions;
+using LT.DigitalOffice.Kernel.BrokerSupport.Helpers;
+using LT.DigitalOffice.Kernel.BrokerSupport.Middlewares.Token;
+using LT.DigitalOffice.Kernel.Configurations;
+using LT.DigitalOffice.Kernel.Extensions;
+using LT.DigitalOffice.Kernel.Helpers;
+using LT.DigitalOffice.Kernel.Middlewares.ApiInformation;
+using LT.DigitalOffice.Kernel.RedisSupport.Configurations;
+using MassTransit;
+using MassTransit.RabbitMqTransport;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Serilog;
+using StackExchange.Redis;
+using System;
+using LT.DigitalOffice.WikiService.Models.Dto.Configurations;
+
+namespace LT.DigitalOffice.WikiService
+{
+  public class Startup : BaseApiInfo
+  {
+    public const string CorsPolicyName = "LtDoCorsPolicy";
+    private string redisConnStr;
+
+    private readonly BaseServiceInfoConfig _serviceInfoConfig;
+    private readonly RabbitMqConfig _rabbitMqConfig;
+
+    public IConfiguration Configuration { get; }
+
+    #region public methods
+
+    public Startup(IConfiguration configuration)
+    {
+      Configuration = configuration;
+
+      _serviceInfoConfig = Configuration
+        .GetSection(BaseServiceInfoConfig.SectionName)
+        .Get<BaseServiceInfoConfig>();
+
+      _rabbitMqConfig = Configuration
+        .GetSection(BaseRabbitMqConfig.SectionName)
+        .Get<RabbitMqConfig>();
+
+      Version = "1.4.5.2";
+      Description = "UserService is an API that intended to work with users.";
+      StartTime = DateTime.UtcNow;
+      ApiName = $"LT Digital Office - {_serviceInfoConfig.Name}";
+    }
+    public void ConfigureServices(IServiceCollection services)
+    {
+      services.AddCors(options =>
+      {
+        options.AddPolicy(
+          CorsPolicyName,
+          builder =>
+          {
+            builder
+              .AllowAnyOrigin()
+              .AllowAnyHeader()
+              .AllowAnyMethod();
+          });
+      });
+
+      string connStr = Environment.GetEnvironmentVariable("ConnectionString");
+      if (string.IsNullOrEmpty(connStr))
+      {
+        connStr = Configuration.GetConnectionString("SQLConnectionString");
+
+        Log.Information($"SQL connection string from appsettings.json was used. Value '{PasswordHider.Hide(connStr)}'.");
+      }
+      else
+      {
+        Log.Information($"SQL connection string from environment was used. Value '{PasswordHider.Hide(connStr)}'.");
+      }
+
+      services.AddHttpContextAccessor();
+
+      services.AddHealthChecks()
+        .AddRabbitMqCheck()
+        .AddSqlServer(connStr);
+
+      /*services.AddDbContext<WikiServiceDbContext>(options =>
```

Comment:
I guess you need add interface IDataProvider and class WikiServiceDbContext and uncomment this and lines 172, 174

Reply:
Yes, it's a plan! Just for now interface IDataProvider and class WikiServiceDbContext does not exist. When they appear, this lines (85-88, 172, 174) would be uncommented.